pkg/util/mount/mount.go

```diff
@@ -274,7 +279,7 @@ func IsNotMountPoint(mounter Interface, file string) (bool, error) {
 		return notMnt, mountPointsErr
 	}
 	for _, mp := range mountPoints {
-		if mounter.IsMountPointMatch(mp, file) {
+		if mounter.IsMountPointMatch(mp, resolvedFile) {
```

Comment:
err - tests may be?

Reply:
That's the problematic part. e2e test would require changing /var/lib/kubelet into a symlink somehow.

Reply:
Can we not write/update unit test for `IsNotMountPoint`, so as it considers symbolic links properly?

Reply:
Proper IsNotMountPoint unit test would require mounting / bind-mounting stuff, which is impossible in an unit test. I could fake all this, but then it would just test very little piece of code and I don't think it's worth all the effort.